fpm/src/fpm/cmd/new.f90

```diff
@@ -0,0 +1,180 @@
+        &'contains',                              &
+        &'  subroutine say_hello',                &
+        &'    print *, "Hello, '//bname//'!"',    &
+        &'  end subroutine say_hello',            &
+        &'end module '//bname]
+       ! a proposed alternative default
+        call warnwrite(join_path(settings%name, 'src', bname//'.f90'), littlefile) ! create NAME/src/NAME.f90
+    endif
+
+    if(settings%with_test)then
+        call mkdir(join_path(settings%name, 'test'))                            ! create NAME/test or stop
+        message=[character(len=80) ::  message,   &                             ! create next section of fpm.toml
+        &'[[test]]                             ', &
+        &'name="runTests"                      ', &
+        &'source-dir="test"                    ', &
+        &'main="main.f90"                      ', &
+        &'']
+
+        littlefile=[character(len=80) ::        &
+        &'program main',                       &
+        &'implicit none',                      &
+        &'',                                   &
+        &'print *, "Put some tests in here!"', &
+        &'end program main']
+        ! a proposed alternative default a little more substantive
+        call warnwrite(join_path(settings%name, 'test/main.f90'), littlefile)    ! create NAME/test/main.f90
+    endif
+
+    if(settings%with_executable)then
+        call mkdir(join_path(settings%name, 'app'))                             ! create NAME/app or stop
+        message=[character(len=80) ::  message,   &                             ! create next section of fpm.toml
+        &'[[executable]]                       ', &
+        &'name="'//bname//'"                   ', &
+        &'source-dir="app"                     ', &
+        &'main="main.f90"                      ', &
+        &'']
+
+       
+        if(exists(bname//'/src/'))then
+            littlefile=[character(len=80) ::          &
+            &'program main',                          &
+            &'  use '//bname//', only: say_hello',    &
+            &'  implicit none',                       &
+            &'',                                      &
+            &'  call say_hello()',                    &
+            &'end program main']
+        else
+            littlefile=[character(len=80) ::                 &
+            &'program main',                                 &
+            &'  implicit none',                              &
+            &'',                                             &
+            &'  print *, "hello from project '//bname//'"',  &
+            &'end program main']
+        endif
+        call warnwrite(join_path(settings%name, 'app/main.f90'), littlefile)
+    endif
+    call warnwrite(join_path(settings%name, 'fpm.toml'), message)               ! now that built it write NAME/fpm.toml
+
+    call run('cd ' // settings%name // ';git init')    ! assumes these commands work on all systems and git(1) is installed
```

Comment:
I believe the semicolon here is the cause of the `git init` failures on Windows
```suggestion
    call run('cd ' // settings%name // '&&git init')    ! assumes these commands work on all systems and git(1) is installed
```

A quick test shows that replacing with double ampersand should work cross platform.

Reply:
Great. Thanks.semi-colon does not work on MSWiindows?

Reply:
Why not just `git init <dir>`? There is no need to change the directory for this git command.

Reply:
entering the directory helps ensure it is created as the mkdir is slightly inconsistent across platforms. It sometimes stops the program if it fails on MSWindows and does not on GNU/Unix systems. After testing how git performs on a non-exist directory it looks like `git init <dir>` would work as well and is simpler. See above for discussion on making list_files() and mkdir() a little more consistent across platforms. I will make that change.